jstest/__main__.py

```diff
@@ -144,7 +148,14 @@ def adjust_options(options):
         options.no_flash = True
 
         if options.device in ['rpi2', 'artik530']:
-            options.no_test = True
+            options.username = 'js-remote-test'
+            options.password = 'jerry'
+            options.ip = '127.0.0.1'
+            options.port = 2022
+            options.remote_workdir = 'emulated_workdir'
```

Comment:
Unnecessary overwrite.

Reply:
If we don't overwrite `remote_workdir`, we will get an error message:
`[SystemError] Please use --remote-workdir for the device.`
This error message comes from the `SSHDevice` class, which should not distinguish if we are using an emulated device or a real device, so overwriting `remote_workdir` is necessary.